e2e/container/run_test.go

```diff
@@ -25,6 +28,8 @@ func TestRunAttachedFromRemoteImageAndRemove(t *testing.T) {
 }
 
 func TestRunWithContentTrust(t *testing.T) {
+	environment.SkipIfRemoteDaemon(t)
```

Comment:
nit: Just to look a bit more inline with what we do on other part (in moby or else) : 

```go
skip.If(t, environment.RemoteDaemon())
```

Reply:
The reason I did it like this was to have the same skip message if we're skipping for the same reason.

I guess I could do `skip.If(t, environment.RemoteDaemon(), environment.RemoteDaemonSkipMessage)`

wdyt?